shae-riches/week7/mon/js/main.js

```diff
@@ -20,22 +20,19 @@ const queryMovieDatabase = function( movieTitle ) {
         $(`#results`)
             .empty()    //                  remove previous results    
             .prepend(`
-                <h2>'${ '#movieSearchInput' }' Search Results</h3>
-                ${ for (let i = 0; i < array.length; i++) {
-                    const element = array[i];
-                    
-                    }
+                <h2>'${ `${movieSearchInput}` }' Search Results</h3>
+                
 
-
-                }
                 <h4>${ data }</h4>
 
-            `);         //                  output of queryData to index.html page as #results div
- 
+            
```

Comment:
`console.log(data)` to see what this returns (after you h ave removed `?json`)
You should then handle displaying your data in a different function. 

You have the first two things complete:
- Getting the user input 
- querying the database - fetching the API info with user input.

Now you need functions to:
- handle the result (display the data) - start with `displayData(data)` on line 18 and write a new function below called `displayData( info ) {}` and `console.log(info)` to see what you get and how you would then access the information and display it with `.append()` etc...
-